NEW_README.md

```diff
@@ -141,15 +140,155 @@ Run prints its output if either,
 2. no tasks are "long" (eg a one-shot "build" procedure, rather than an ongoing
    "dev server").
 
+## Task Configuration Details
+
+The core of Run's configuration is the `tasks.toml` file, which defines the tasks you want Run to manage. Each task is specified with a range of properties, as shown in the example below:
+
+```toml
```

Comment:
reviewer note: this is the existing kitchen-sink example

Reply:
todo: "simulate-coding" is a weird task name (here adopted from our example project, which is, itself, very weird)

since this isn't meant to be valid anyway (it specifies deps and triggers that don't exist) it would be better to make the tasks realistic (eg "while true do sleep 1 done" is not a realistic dev cmd)